src/main/java/uk/gov/hmcts/reform/blobrouter/config/StorageConfiguration.java

```diff
@@ -32,11 +32,10 @@ public LeaseClientProvider getLeaseClientProvider() {
     @Bean
     public BlobServiceClient getStorageClient(
         StorageSharedKeyCredential credentials,
-        @Value("${storage.url}") String storageUrl
+        @Value("${storage.reform.connection-string}") String connectionString
```

Comment:
how will that work for all the other environments?

Reply:
for aat and  prod we will use external url which will be bootstrap from vault

Reply:
in that case, we need all the usual recipes for value: chart versions, values template for all envs, flux config 😉

Reply:
I will add values to vault and revert code changes and release this.